### PR TITLE
Feature/roadmap

### PR DESCRIPTION
**Pull request by PabloAbara**
Uses roadmap controller to get or create the first Roadmap, and its first Checkpoint. Then, it shows both on Roadmap/index view.

### CODE REVIEW

File: app/controllers/roadmap_controller.rb [hunk 1]
```diff
@@ -2,5 +2,7 @@ class RoadmapController < ApplicationController
   before_action :authenticate_user!
   
   def index
+    @roadmap = Roadmap.get_or_Create()
```
Comment (isavega) [inline app/controllers/roadmap_controller.rb:5]: Borrar método get_or_Create() y agregar seeds

File: app/models/checkpoint.rb [hunk 2]
```diff
@@ -4,4 +4,14 @@ class Checkpoint < ApplicationRecord
   has_many :completed_checkpoints
   has_many :users, through: :completed_checkpoints  
 
+
+  def self.get_or_Create(roadmap)
```
Comment (isavega) [inline app/models/checkpoint.rb:8]: Eliminar este método

pd: no mezclar snake_case con camelCase

File: app/models/roadmap.rb [hunk 3]
```diff
@@ -1,3 +1,13 @@
 class Roadmap < ApplicationRecord
   has_many :checkpoints
-end
+
+  def self.get_or_Create()
```
Comment (isavega) [inline app/models/roadmap.rb:4]: Lo mismo, reemplazar por seeds

File: app/controllers/roadmap_controller.rb (hunk 1 above)
Comment (PabloAbara) [reply app/controllers/roadmap_controller.rb:5]: listo!

File: app/models/checkpoint.rb (hunk 2 above)
Comment (PabloAbara) [reply app/models/checkpoint.rb:8]: :ok

**Pull request merged**